src/controller/CHIPDevice.h

```diff
@@ -330,6 +328,10 @@ class DLL_EXPORT Device : public Messaging::ExchangeDelegate
     void CancelResponseHandler(uint8_t seqNum);
     void AddReportHandler(EndpointId endpoint, ClusterId cluster, AttributeId attribute, Callback::Cancelable * onReportCallback);
 
+    // This two functions are pretty tricky, it is used to bridge the response
```

Comment:
It is not clear to me that this comment brings anything really meaningful beside the fact that you do not like them :)

Reply:
Just updated the comments.